gnucash/gnome/gnc-plugin-page-report.c

```diff
@@ -1140,6 +1141,7 @@ static action_toolbar_labels toolbar_labels[] =
        to be used as toolbar button label. */
     { "ReportSaveAsAction", N_("Save Config As...") },
     { "FilePrintPDFAction", N_("Make Pdf") },
+    { "OpenExternalBrowserAction", N_("External") },
```

Comment:
"External" is a bit too terse even in English; it at least needs a context and maybe a translator comment to explain it.

Reply:
What would be more ideal ?

Reply:
"Open in External Browser".